### PR TITLE
MongoDB AWS backup script

### PR DESCRIPTION
**Pull request by juhhov**
## Status
**READY**

## Description
Take a database dump from mongodb and push it to AWS S3 bucket.

### CODE REVIEW

File: scripts/aws_backup.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+HOST=localhost
+
+# DB name
+DBNAME=db_name
+
+# S3 bucket name
+BUCKET=s3_bucket
+
+# Current time
+TIME=`/bin/date --iso-8601=seconds`
```
Comment (jupe) [inline scripts/aws_backup.sh]: maybe more human readable and sortable format: 
`date -u +"%Y-%m-%dT%H%M%SZ"`

File: scripts/aws_backup.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Prerequisites:
+# 1. Configured aws client, new enough version needed
+# 2. S3 bucket
```
Comment (jupe) [inline scripts/aws_backup.sh]: add some link/description where to get/configure aws client

File: scripts/aws_backup.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+#
+# Restore with command:
+# mongorestore --gzip --archive < <backup.gzip>
+
+set -e
+
+begin=$(date +%s)
+
+# DB host (secondary preferred as to avoid impacting primary performance)
+HOST=localhost
```
Comment (jupe) [inline scripts/aws_backup.sh]: add these as script arguments so user doesn't need to modify this script...

File: scripts/aws_backup.sh (hunk 1 above)
Comment (jupe) [inline scripts/aws_backup.sh]: maybe abs path is not needed for these tools - they probably are in path already.. could be even different in some distros..

Comment (juhhov) [inline scripts/aws_backup.sh]: ISO 8601 is sortable and also very human readable.

Comment (jupe) [inline scripts/aws_backup.sh]: ah true 👌

**Pull request merged**